.travis.yml

```diff
@@ -106,6 +106,9 @@ matrix:
       env: "DB=mysql SAUCE=TRUE SELENIUM_BROWSER=firefox"
     - php: 7.2
       env: "DB=mysql SAUCE=TRUE SELENIUM_BROWSER=chrome"
-
+    - php: 7.0
+      env: "DB=mysql CORE_BRANCH=stable15"
+    - php: 7.2
+      env: "DB=mysql CORE_BRANCH=v15.0.0RC1"
```

Comment:
not just stable15?

Reply:
I'll change that to the .0 release once tagged to verify the app works with all minor/patch versions of 15 and not just the unreleased stable branch.